platform/src/main/java/org/stellar/anchor/platform/payment/observer/stellar/StellarPaymentObserver.java

```diff
@@ -98,7 +112,7 @@ String fetchStreamingCursor() {
     return pageOpResponse.getRecords().get(0).getPagingToken();
   }
 
-  public SSEStream<OperationResponse> watch() {
+  private SSEStream<OperationResponse> watch() {
```

Comment:
`private` can make mocking difficult. Can we use package private?

Reply:
Alright, I can remove the public/private qualifier so it becomes package private.

Reply:
Updated in https://github.com/stellar/java-stellar-anchor-sdk/pull/607/commits/247705aab4bb5e0d048fba7d6ae2d9c768103249.